armi/tests/tutorials/c5g7-settings.yaml

```diff
@@ -1,16 +1,23 @@
 settings:
+# global
   availabilityFactor: 0.9
-  fakeBad: 123456
-  power: 1000000000.0
-  cycleLength: 411.11
-  loadingFile: c5g7-blueprints.yaml
-  nCycles: 10
-  burnSteps: 2
   buGroups:
     - 100
+  burnSteps: 2
   comment: C5G7 LWR Benchmark inputs
-  genXS: Neutron
+  cycleLength: 411.11
+  loadingFile: c5g7-blueprints.yaml
+  nCycles: 10
   numProcessors: 1
-  genReports: false
+  power: 1000000000.0
   versions:
     armi: uncontrolled
+
+# neutronics
+  genXS: Neutron
+
+# report
+  genReports: false
+
+# unknown
+  fakeBad: 123456
```

Comment:
Can we remove this?

Reply:
Hmm... I mean, this is a unit tests file, and that is in the unit test.

I can look at re-writing that test.

Reply:
Okay, I changed the test. We're good.